lib/cookies.js

```diff
@@ -91,7 +91,7 @@ Cookies.prototype.set = function(name, value, opts) {
 
   if (opts && signed) {
     if (!this.keys) throw new Error('.keys required for signed cookies');
-    cookie.value = this.keys.sign(cookie.toString())
+    cookie.value = this.keys.sign(cookie.toString()).toString('base64')
```

Comment:
What are you talking about? This module already installs 1.0.1 by default (since the package.json is ~1.0.0) and it returns a string from the sign method: https://github.com/crypto-utils/keygrip/blob/1.0.1/index.js#L12-L21